### ISSUE TITLE
Version 3.2.2 - App uses your device's current time zone when making a new observation

### ISSUE THREAD

**Issue by tiwane** [inaturalist/INaturalistIOS#621]
1. Import a photo that was taken in a different time zone than the one your device is currently set to. If you need to, manually change your device's time zone to create this mismatch.
2. Save the observation made with the photo.
3. Look at that observaiton on the website. It's using the time zone to which your device was set when the observation was made.

This also happens when you make a new observation in the app and one's account's time zone setting doesn't affect it.

Expected behavior: when importing a photo, the app should use the photo's time zone (if available) when creating an observation.

**Comment by tiwane** [comment]
> This also happens when you make a new observation in the app and one's account's time zone setting doesn't affect it.

I'm not sure this is the case any more. I just changed my iNat account's time zone to HST on the website. Then in the iOS app (3.2.3), I just made a new observation at 2:37 pm PST here in California. On the website, I see 2:37pm HST:

<img width="509" alt="image" src="https://user-images.githubusercontent.com/15896181/144326113-8913291c-965b-4a75-bd61-731cf2b02d22.png">

Not sure if this is happening on the app side or server side.
